packages/react-core/package.json

```diff
@@ -40,7 +40,7 @@
   "homepage": "https://github.com/patternfly/patternfly-react#readme",
   "scripts": {
     "build:umd": "rollup -c --environment IS_PRODUCTION",
-    "clean": "rimraf dist",
+    "clean": "rimraf dist components layouts helpers next deprecated node_modules",
```

Comment:
This does not pertain to this exact change, but something noticed when running `clean`, I believe these built sub-directories should be included.